app/routers/analytics.py

```
@@ -0,0 +1,36 @@
+from datetime import datetime
+from typing import Dict
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)